packages/config/eslint/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "eslint-config-agreeto",
```

Comment:
So will ESLint resolve this even properly even if we name it `packages/config`?

> [ESLint](https://eslint.org/) resolves configuration files by looking for workspaces with the name eslint-config-*. This lets us write extends: ['custom'] and have ESLint find our local workspace.

[turborepo docs](https://turbo.build/repo/docs/getting-started/create-new#understanding-eslint-config-custom:~:text=ESLint%20resolves%20configuration%20files%20by%20looking%20for%20workspaces%20with%20the%20name%20eslint%2Dconfig%2D*.%20This%20lets%20us%20write%20extends%3A%20%5B%27custom%27%5D%20and%20have%20ESLint%20find%20our%20local%20workspace.)


That'd be nice.

Reply:
The package name is still `eslint-config-agreeto`, doesn't matter if it's nested one level deeper in the directory hierarchy

Reply:
Yeah, it expects a trialing `-` I think.
👍